package.json

```diff
@@ -54,15 +54,15 @@
     "@babel/core": "^7.11.6",
     "@babel/plugin-syntax-dynamic-import": "^7.2.0",
     "@babel/preset-env": "^7.11.5",
-    "@storybook/addon-a11y": "^6.1.11",
-    "@storybook/addon-actions": "^6.1.11",
-    "@storybook/addon-docs": "^6.1.11",
-    "@storybook/addon-knobs": "^6.1.11",
-    "@storybook/addon-viewport": "^6.1.11",
-    "@storybook/components": "^6.1.11",
-    "@storybook/html": "^6.1.11",
-    "@storybook/theming": "^6.1.11",
-    "@storybook/web-components": "^6.1.11",
+    "@storybook/addon-a11y": "^6.1.16",
+    "@storybook/addon-actions": "^6.1.16",
+    "@storybook/addon-docs": "^6.1.16",
+    "@storybook/addon-knobs": "^6.1.16",
+    "@storybook/addon-viewport": "^6.1.16",
+    "@storybook/components": "^6.1.16",
+    "@storybook/html": "^6.1.16",
+    "@storybook/theming": "^6.1.16",
+    "@storybook/web-components": "^6.1.16",
```

Comment:
We should not have both `@storybook/html` and `@storybook/web-components`

We need to drop `@storybook/html` and update the import in : https://github.com/gravitee-io/gravitee-ui-components/blob/cf948b0137340878a8ceeee22787d9af4793432d/stories/lib/i18n-knob.js#L20

It will fix the fact all events are emitted twice 🎉  and surely improve a bit the perf 🤷🏻

Reply:
Done in https://github.com/gravitee-io/gravitee-ui-components/pull/268